exercises/concept/welcome-to-tech-palace/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+There is an appliance store called "Tech Palace" nearby. The owner of the store recently installed a big display to use for marketing messages and to show a special greetings when customers scan their loyalty cards at the entrance. The display consists of lots of small LED lights and can show multiple lines of text.
+
+The store owner needs your help with the code that is used to generate the text for the new display.
+
+## 1. Create the welcome message
+
+For most customers who scan their loyalty cards, the store owner wants to see `Welcome to the Tech Palace, ` following by the name of the customer in capital letters on the display.
+
+Implement the function `WelcomeMessage` that accepts the name of the customer as string as an argument and returns the desired message.
+
+```go
+WelcomeMessage("Judy")
+// Output: Welcome to the Tech Palace, JUDY
+```
+
+## 2. Add a fancy border
+
+For loyal customers that buy a lot at the store, the owner wants the welcome display to be more fancy by adding a line of stars before and after the welcome message. They are not sure yet how many stars should be in the lines so they want that to be configurable.
+
+Write a function `AddBorder` that accepts a welcome message (a string) and the number of stars per line (type `int`) as arguments. It should return a string that consists of 3 lines, a line with the desired number of stars, then the welcome message as it was passed in, then another line of stars.
```

Comment:
```suggestion
Write a function `AddBorder` that accepts a welcome message (a `string`) and the number of stars per line (type `int`) as arguments.
It should return a `string` that consists of 3 lines, a line with the desired number of stars, then the welcome message as it was passed in, then another line of stars.
```

We should probably be consistent if we are going to use backticks around types.

Reply:
Agreed